File: Source code for book/Chapter2/Example01/BugExplorer/BugExplorer/Models/Bug.cs
using System;
using System.Collections.Generic;

namespace BugExplorer.Models
{
    public class Bug
    {
        public int ID { get; set; }
        public string CommonName { get; set; }
        public string Class { get; set; }
        public string Order { get; set; }
        public string Family { get; set; }
        public string Genus { get; set; }
        public string Species { get; set; }
        public string Description { get; set; }
    }
}

File: Source code for book/Chapter2/Example03/Collector/Collector/ClientApp/app/shared/Bug.ts
export class Bug {

    CommonName: string;
    Class: string;
    Order: string;
    Family: string;
    Genus: string;
    Species: string;
    Description: string;
    Filename: string;

    constructor(
        commonName: string,
        className: string,
        order: string,
        family: string,
        genus: string,
        species: string,
        description: string,
        filename: string
    ) {
        this.CommonName = commonName;
        this.Class = className;
        this.Order = order;
        this.Family = family;
        this.Genus = genus;
        this.Species = species;
        this.Description = description;
        this.Filename = filename;
    }
}